.github/workflows/csi-proxy.yml

```diff
@@ -0,0 +1,42 @@
+name: csi-proxy
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - hostprocess/csi-proxy/**
```

Comment:
This should be version file only otherwise it will trigger a push on every file change committed

Reply:
updated